repos/system_upgrade/el7toel8/actors/openldapnsstoopenssl/tests/conftest.py

```diff
@@ -0,0 +1,71 @@
+import copy
+import os
+import os.path as ph
+import random
+import shutil
+import string
+import subprocess as sp
```

Comment:
The same as above

Reply:
Does this apply here since this is not part of the actual actor and only used during testing?

Reply:
@keno That's good question. Something what we should discuss in team better. From my point of view, you shouldn't need to use subprocess anywhere and I don't see benefit to use it (except the point the code is writtent ;-) which is not good argument)  in tests now. Honestly, I have feeling that tests are wrong when you need to use it. For me it's kinda destructive tests as it looks like you are affecting the system (I am not familiar with the `certutil` utility, so I can be wrong). But why you need it? If I understand right, you can replace certutil completely (I just looked at the code very roughly).

1) You do not need to generate certificates (those can be added in tests/files or files/tests).

2) For output given by certutil, you can return expected strings directly to the actor (mock the run function).

But as I told, I looked at it in very short time, so I could miss something, so take this comment with discretion. We have already one case (selinux) where we accepted destructive tests because there was not simple way to write test in the safe way. Such tests has to be "treated" so these are skipped by default.